x-pack/plugins/canvas/public/components/workpad_header/control_settings/auto_refresh_controls.js

```diff
@@ -61,9 +62,7 @@ export class AutoRefreshControls extends Component {
             </EuiDescriptionList>
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
-            <EuiButton size="s" fill iconType="refresh" onClick={doRefresh} isDisabled={inFlight}>
-              Refresh
-            </EuiButton>
+            <RefreshControl />
```

Comment:
The refresh button in the popover is an intentional duplication, since we plan to swap out that control based on the refresh state; see https://github.com/elastic/kibana/issues/35849.